autoload/ale/python.vim

```diff
@@ -25,8 +25,7 @@ function! ale#python#FindProjectRootIni(buffer) abort
         \|| filereadable(l:path . '/tox.ini')
         \|| filereadable(l:path . '/mypy.ini')
         \|| filereadable(l:path . '/pycodestyle.cfg')
-        \|| filereadable(l:path . '/flake8.cfg')
-        \|| filereadable(l:path . '/.flake8rc')
+        \|| filereadable(l:path . '/.flake8')
```

Comment:
You can move this further up the list, as this is the officially supported filename.